csatar-plugins/forms/assets/css/style.css

```diff
@@ -0,0 +1,33 @@
+.fs-header {
+	padding: 3% 5% 6% 5%;
+	background-color: slategrey;
+}
```

Comment:
Ebből a CSS-ből csak a legfontosabbakat hagynám meg.
Színek semmiképp és a méretezések közül is CSAK azokat amelyek föltétlen szükségesek